client/composables/useTheme.ts

```diff
@@ -7,10 +7,6 @@ export default function useTheme() {
         rel: 'stylesheet',
         href: isDark ? darkTheme : lightTheme,
```

Comment:
might as well remove the $ on line 2 and :

```suggestion
        href: isDark.value ? darkTheme : lightTheme,
```